docs/src/development/local/_index.md

```diff
@@ -1,151 +1,92 @@
-Each new environment is a child of the environment which you created it from.
-Child environments can sync code and/or data down from their parent environment. 
-They can also merge code up to their parent, 
-making them a good place for development, staging, and testing.
-
-Before downloading a copy of your code,
-you might want to view a list of all the environments in your project to pick one.
-
-To do so, run the following command:
-
-```bash
-platform environments 
-```
-
-You get output similar to the following:
-
-```bash
-Your environments are: 
-+--------------------+------------------+--------+-------------+
-| ID                 | Title            | Status | Type        |
-+--------------------+------------------+--------+-------------+
-| main               | Main             | Active | production  |
-|   march-release    | march release    | Active | development |
-|   march-tests      | march tests      | Active | development |
-|   april-release    | april release    | Active | development |
-|   april-tests      | april tests      | Active | development |
-+--------------------+------------------+--------+-------------+
-```
-
-### Launch the download
-
-If your app's code isn't already present locally on your computer,
-you can download a copy of it.
-To do so, move to the directory where you want to save a copy of your project 
-and run this command:
-
-```bash
-platform get {{< variable "PROJECT_ID" >}}
-```
-
-When prompted, choose a name for the target project directory and press **Enter**.
-You get output similar to the following:
-
-```
-Downloading [project-name](abcdefgh1234567)
-  Cloning into '/Users/username/project-directory'...
-  remote: counting objects: 3318, done.        
-  Receiving objects: 100% (3318/3318), 20.23 MiB | 3.71 MiB/s, done.
-  Resolving deltas: 100% (568/568), done.
-
-The project [project-id](abcdefgh1234567) was successfully downloaded to: 'project-directory'
-```
-
-You can now access your app's code from the project directory created on your computer.
-Note that inside the project directory, the `.platform/local` subdirectory is excluded from Git. 
-It contains builds and any local metadata about your project the [Platform.sh CLI](../../administration/cli/_index.md) needs.
+You need to have:
+
+- A Platform.sh account:
+  new users can sign up for a [free trial account](https://auth.api.platform.sh/register)
+- A working project,
+  either started from a [template](../../development/templates.md) 
+  or with your own code pushed to Platform.sh
+- [Git](https://git-scm.com/downloads)
+- The [Platform.sh CLI](../../administration/cli/_index.md)
+
+## 1. Get your code
+
+If you don't have your app code on your computer, download a copy.
+
+1.  Get your project ID by running `platform projects`.
+
+2.  Get the code by running the following command:
+
+    ```bash
+    platform get {{< variable "PROJECT_ID" >}} {{< variable "TARGET_DIRECTORY_NAME" >}}
```

Comment:
I'm gradually getting used to using `platform get` but since just using git is a for more general technique, would it be appropriate to lead with that? Or offer it as alternative?

Reply:
It's certainly possible. It seemed like it would add complexity if we had to add the steps to get the Git command, but something to consider, for sure.